app/src/main/java/com/example/travellens/Camera.java

```diff
@@ -0,0 +1,59 @@
+package com.example.travellens;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.database.Cursor;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.ImageDecoder;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.core.content.FileProvider;
+import androidx.fragment.app.Fragment;
+
+import com.parse.ParseFile;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Camera {
+    private Context context;
+    private Activity activity;
```

Comment:
You normally want to avoid storing things like Activity and Context as member variables like this.

In this case, you could just pass Context as parameter in getPhotoFileUri(). I don't see Activity being used at all.